docs/github-webhook.md

```diff
@@ -0,0 +1,109 @@
+
+4. Optionally, select the Content type drop-down menu, and click a data format to receive the webhook payload in. For this case `application/json` is needed.
+
+![config payload url](./configuration//github_webhook/config_payload_url.png)
+
+5. Under "Which events would you like to trigger this webhook?", select the webhook events that you want to receive. You should only subscribe to the webhook events that you need in your app. In this case the `Let me select individual events.` option will be enabled to select only the 'Pull request' events.
+
+6. To make the webhook active immediately after adding the configuration, select Active and Click `Add webhook` button.
+
+![add webhook](./configuration/github_webhook/add_webhook_button.png)
+
+### 3. Setup Github PAT
+
+Personal access tokens are an alternative to using passwords for authentication to GitHub when using the GitHub API or the command line. Personal access tokens are intended to access GitHub resources on behalf of yourself.
+
+> [!NOTE]
+> if you want detailed information about this integration process you could check this [documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)
+
+1. In the upper-right corner of any page on GitHub, click your profile photo, then click Settings.
+   ![profile settings](./configuration/github_pat/profile_settings.png)
+
+2. In the left sidebar, click Developer settings.
+   ![developer settings](./configuration/github_pat/developer_settings.png)
+
+3. In the left sidebar, under PAT tab, click Tokens (classic). Then click `Generate new token` (classic).
+   ![classic token](./configuration/github_pat/classic_token.png)
+
+4. In the "Note" field, give your token a descriptive name.
+
+5. To give your token an expiration, select Expiration, then choose a default option or click Custom to enter a date.
+
+6. Select the scopes you'd like to grant this token. To use your token to access repositories from the command line, select repo. A token with no assigned scopes can only access public information. For more information, see Scopes for OAuth apps.
+
+![pat_classic_form](./configuration/github_pat/pat_classic_form.png)
+
+7. Click Generate token. So the token is available for your app now!
+
+### 4. Finalize Discord Bot Configuration
+
+Once the different configurations have been made to connect discord with the github webhook, some of the data obtained is taken to finish with the configuration and integration of the bot.
+
+#### Steps:
+
+1. Configure the PAT obtained from the github profile in the environment variable `GITHUB_ACCESS_TOKEN`
+
+2. The channel that will receive the open PR alert messages is configured by means of the environment variable `GITHUB_PR_REVIEW_CHANNEL`
```

Comment:
GITHUB_PR_PUBLISH_ENABLED is another .env that should be updated to enable the feature, should we mention it here?
![2025-03-31-09:00:29-screenshot](https://github.com/user-attachments/assets/12a8e4a1-7708-448b-8f8e-e7f6b21c8acc)